tensorrt_llm/__init__.py

```
@@ -50,7 +50,7 @@ def _add_trt_llm_dll_directory():
 from .builder import BuildConfig, Builder, BuilderConfig, build
 from .disaggregated_params import DisaggregatedParams
 from .functional import Tensor, constant
-from .llmapi import LLM, LlmArgs
+from .llmapi import LLM, LlmArgs, MultimodalEncoder
```

Ruff (0.12.2)

Module level import not at top of file

(E402)